GVFS/GVFS.Common/Prefetch/Git/DiffHelper.cs

```diff
@@ -31,7 +32,8 @@ public DiffHelper(ITracer tracer, Enlistment enlistment, IEnumerable<string> fil
         public DiffHelper(ITracer tracer, Enlistment enlistment, GitProcess git, IEnumerable<string> fileList, IEnumerable<string> folderList, bool includeSymLinks)
         {
             this.tracer = tracer;
-            this.fileList = new List<string>(fileList);
+            this.exactFileList = new HashSet<string>(fileList.Where(x => !x.StartsWith("*")).Select(x => x.ToLower()));

```

Comment:
Should use `new HashSet<string>(fileList.Where(x => !x.StartsWith("*")), StringComparer.OrdinalIgnoreCase)` instead of having to use `ToLower()` here and in `Contains` below.

Reply:
Good point, changed.